atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/SimpleDbWriteTable.java

```diff
@@ -64,8 +68,8 @@ public void put(Collection<Map.Entry<Cell, Value>> data) {
     private void put(List<Object[]> args) {
         try {
             conns.get().insertManyUnregisteredQuery(
-                    "/* SQL_MET_INSERT_ONE (" + tableName + ") */" +
-                    " INSERT INTO pt_met_" + tableName + " (row_name, col_name, ts, val) " +
+                    "/* INSERT_ONE (" + tableName + ") */" +
```

Comment:
Is changing this comment going to cause problems for log reading tools?  I'm assume it's there to make tracing / monitoring sql commands easier.